### PR TITLE
[peertube] implement sepia search

### PR DESCRIPTION
**Pull request by B0pol**
#### What is it?
- [ ] Bugfix (user facing)
- [x] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Implement sepia search, Framasoft's index

#### Fixes the following issue(s)

- #4461

#### Relies on the following changes

- teamnewpipe/newpipeextractor#495

#### APK testing 

use the ci

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by HTSamurai** [comment]
sounds good

**Review by TobiGr — COMMENTED** [review]
Looks good to me.

Can be merged once the extractor PR is merged and the dependency in `build.gradle` is updated.

**Comment by B0pol** [comment]
done @TobiGr

**Review by TobiGr — APPROVED** [review]

**Pull request merged**
